cerberus/tests/conftest.py

```diff
@@ -1,12 +1,49 @@
 # -*- coding: utf-8 -*-
 
 from copy import deepcopy
+from pkg_resources import Distribution, DistributionNotFound
 
 import pytest
 
 from cerberus import Validator
 
 
+def reload_module(name):
+    try:
+        reload
+    except NameError:
+        try:
+            from importlib import reload
+        except ImportError:
+            from imp import reload
+    reload(name)
+
+
+@pytest.fixture
+def cerberus():
+    return __import__('cerberus')
+
+
+@pytest.fixture(scope="function")
```

Comment:
as these fixtures aren't reused, i think the monkeypatching should happen in the test functions to make it more obvious what it is about and what's happening. is the `cerberus` fixture even necessary with that style?

Reply:
I guess you are right and we should monkeypatch inside the test function. Might be clearer to read.